web/GenerateIcs.php

```diff
@@ -203,7 +203,7 @@ function createIcsFile($ociIds, $ociIdData, $season, $outputStream)
     $ociId = (int) $ociId;
 }
 
-$netId = ProjectCommon::casAuthenticate(true);
+$netId = ProjectCommon::getID();
```

Comment:
`casAuthenticate(true)` previously would redirect if the user wasn't logged in, whereas `casAuthenticate(false)` would return null when the user wasn't logged in. I don't think this will cause any issues, but you probably know better than I do.

Reply:
yeah, you're right and I don't think any of the calls casAuthenticate(true) actually get called so it shouldn't be a problem, Only Table.php and GenerateIcs.php ever call it, which I don't think are used (?), but even if they are used I don't think it would be an issue.

Reply:
sounds good, thanks for looking into it